### PR TITLE
Jonfritz patch 2

### PR DESCRIPTION
**Pull request by jonfritz**
Updated the second half of the documentation.

### CODE REVIEW

File: docs/source/tutorials/conversational_memory_with_langchain.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Prepare OpenSearch client for Sycamore
 
-Since the conversation memory API is a new feature in OpenSearch, it isn't yet available in the various language clients. Luckily, it’s super easy to just create it. First, for dependencies: this requires the `opensearchpy` package, OpenSearch’s python client:
+Sycamore is compatible with the OpenSearch conversation memory APIs, and you can use an OpenSearch client to access them. However, these APIs are not yet available in the various language clients at the time we wrote this tutorial. Luckily, it’s super easy to just create it. First, for dependencies: this requires the `opensearchpy` package, OpenSearch’s python client:
```
Comment (HenryL27) [inline docs/source/tutorials/conversational_memory_with_langchain.md:7]: Luckily, it's super easy to create it **in python**

File: docs/source/tutorials/sycamore-jupyter-dev-example.md [hunk 2]
```diff
@@ -301,27 +234,26 @@ st_embed_docset.write.opensearch(os_client_args=os_client_args, index_name=index
...
+2. Once the data is loaded into Sycamore, you can use the [demo query UI](../querying_data/demo_query_ui.md) for conversational search on it.
+- Using your internet browser, visit `http://localhost:3000`. 
+- Make sure the index selected in the dropdown at the bottom of the UI has the same name you provided in step 1j
+- Create a new conversation. Enter the name for your conversation in the text box in the left "Conversations" panel, and hit enter or click the "Add convo" icon on the right of the text box.
 - As a sample question, you can ask "Who wrote Attention Is All You Need?"
 
-The results of the hybrid search are in the right hand panel, and you can click through to find the highlighted passage (step 3b enabled this). Though we are getting good results back from hybrid search, it would be nice if we could have the titles and other information for each passage. In the next section, we will iterate on our Sycamore job, and use generative AI to extract some metadata.
+The results of the hybrid search are in the right hand panel, and you can click through to find the highlighted passage (step 3b enabled this).
```
Comment (HenryL27) [inline docs/source/tutorials/sycamore-jupyter-dev-example.md:246]: Has step 3b occurred yet?

File: docs/source/tutorials/sycamore-jupyter-dev-example.md [hunk 3]
```diff
@@ -301,27 +234,26 @@ st_embed_docset.write.opensearch(os_client_args=os_client_args, index_name=index
...
+
+Though we are getting good results back from hybrid search, it would be nice if we could have the titles and other information for each passage. In the next section, we will iterate on our Sycamore job, and use a Sycamore transform that leverages generative AI to extract some metadata.
 
 ## Add metadata extraction using GenAI
 
-Now we are going to edit our existing notebook to show how you could adjust the Sycamore
-processing.  Because we are editing the notebook, we will be making changes to the cells partway
-through the notebook, and then editing some of the nearby cells to connect the new processing into
-the old processing.
+Next, we are going to edit our existing notebook to add an additional processing step. We will be making changes to the cells partway through the notebook, and then editing some of the nearby cells to connect the new processing steps into the exisitng steps.
```
Comment (HenryL27) [inline docs/source/tutorials/sycamore-jupyter-dev-example.md:252]: existng -> existing

**Review by HenryL27 — COMMENTED** [review]
mostly spelling

File: docs/source/querying_data/integrate_your_application.md [hunk 4]
```diff
@@ -70,7 +70,7 @@ try {
 }
 ```
 
-**********Note:********** By default, all conversations are visible to every client accessing the cluster. In a production setup, you will want to [use access control](https://opensearch.org/docs/latest/security/access-control/index/) to restrict the conversations a user can see.
+*Note: By default, all conversations are visible to every client accessing the cluster. In a production setup, you will want to [use access control](https://opensearch.org/docs/latest/security/access-control/index/) to restrict the conversations a user can see.*
```
[the remark this answers is not in the document]
Comment (HenryL27) [reply docs/source/querying_data/integrate_your_application.md:73]: ( thats: "> Note: blah blah" )

File: docs/source/conversation_memory/overview.md [hunk 5]
```diff
@@ -2,9 +2,9 @@
...
 
-You can also use Sycamore's conversion memory feature set as a scalable conversation storage for generative AI agenets and other applications. It also makes it easy to debug and tune your LLM applications by saving the prompt and response from the LLM. When prompt engineering, tuning, and debugging an LLM-based app, this is critical information - as the exact wording of a prompt can grealty effect the response.
+You can also use Sycamore's conversion memory feature set directly as a scalable conversation storage for generative AI agenets and other applications. It also makes it easy to debug and tune your LLM applications by saving the prompt and response from the LLM. When prompt engineering, tuning, and debugging an LLM-based app, this is critical information - as the exact wording of a prompt can grealty effect the response.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/conversation_memory/overview.md:7]: Ack

File: docs/source/conversation_memory/using_with_conversational_search.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ The RAG pipeline will retrieve up to the last 10 of interactions from the conver
 
 ## Question rewriting
 
-We recommend using "question rewriting" when building a conversational search application, which also requires conversation memory. Question rewriting takes a user quesiton submitted to the application and changes it to reflect the context of the conversation. This can make the question more clear for Sycamore to return a better answer, especially if the question is vague or requires context from a prior interaction (e.g. using prepositions that refer to things in previous interactions). This also creates more of a natural conversational experience.
+We recommend using question rewriting when building a conversational search application, which also requires conversation memory. Question rewriting takes a user quesiton submitted to the application and changes it to reflect the context of the conversation. This can make the question more clear for Sycamore to return a better answer, especially if the question is vague or requires context from a prior interaction (e.g. using prepositions that refer to things in previous interactions). This also creates more of a natural conversational experience.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/conversation_memory/using_with_conversational_search.md:32]: Ack

File: docs/source/querying_data/demo_query_ui.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
 
-Sycamore includes a simple UI to submit queries and display retreival-augmented generation (RAG) answers and hybrid search results. The UI is deployed in the conatiner `sycamore-demo-ui` and is accessed at `localhost:3000`. It is an easy way to test the answer quality from your Sycamore stack, and how iterating with your data preparation and enrichment affects the overal quality.
+Sycamore includes a simple UI to submit queries and display retreival-augmented generation (RAG) answers and hybrid search results. The UI is deployed in the conatiner `sycamore-demo-ui` and is accessed at `http://localhost:3000`. It is an easy way to test the answer quality from your Sycamore stack, and how iterating with your data preparation and enrichment affects the overal quality.
 
-You can create a new conversation by entering the name in the text box in the "Conversations" panel and hit enter. Then, you can submit questions in the middle panel.
+You can create a new conversation by entering the name in the text box in the "Conversations" panel and hit enter. Makes sure the conversation is selected in the left panel. Then, you can submit questions in the middle panel.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/demo_query_ui.md]: Ack

File: docs/source/querying_data/demo_query_ui.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
 
-Screenshot answering questions on financial data:
+The screenshot below shows the demo query UI being used to answer questions on financial data:
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/demo_query_ui.md:7]: Yes - it's a 10Q. Do you think I need to make that clear?

[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/demo_query_ui.md]: Made an update

File: docs/source/querying_data/generative_ai_configurations.md [hunk 9]
```diff
@@ -2,12 +2,8 @@
...
 
-
-
-Amazon Bedrock models and Anthropic are supported as well. To configure these, you can follow these instructions.  NEED INFORMATION
-
-NOTE – will write these when poritng to Git  https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/index/ and https://opensearch.org/docs/latest/search-plugins/conversational-search/#rag-pipeline https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/blueprints/
+Amazon Bedrock models and Anthropic are supported as well. To configure these, you need to configure the OpenSearch components of Sycamore directly. You can [configure the model](https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/index/), [create a connector blueprint]( https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/blueprints/), and [connect it to your RAG pipelines](https://opensearch.org/docs/latest/search-plugins/conversational-search/#rag-pipeline).
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/generative_ai_configurations.md]: Ack

File: docs/source/querying_data/integrate_your_application.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/integrate_your_application.md]: Ack. I didn't know you could do this!

File: docs/source/querying_data/using_rag_pipelines.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Using Retrieval-Augmented Generation (RAG) pipelines
 
 ## Overview
-[Retrieval-augmented generation (RAG)](https://arxiv.org/abs/2005.11401) is a popular method to generate natural language answers to questions using LLMs and indexed data. It retreives relevant data to a query, and then sends it along with a prompt to a LLM to sythensize an answer. Sycamore implements RAG using an [OpenSearch Search Pipeline](https://opensearch.org/docs/latest/search-plugins/conversational-search/#rag-pipeline) to orchestrate interactions with LLMs.
+[Retrieval-augmented generation (RAG)](https://arxiv.org/abs/2005.11401) is a popular method to generate natural language answers to questions using LLMs and indexed data. It retreives relevant data to a query, and then sends it along with a prompt to a LLM to sythensize an answer. Sycamore implements RAG using an [OpenSearch Search Pipeline](https://opensearch.org/docs/latest/search-plugins/conversational-search/#rag-pipeline) to orchestrate interactions with LLMs, and is compatible with those APIs.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/using_rag_pipelines.md]: Ack.

File: docs/source/tutorials/conversational_memory_with_langchain.md (hunk 1 above)
Comment (jonfritz) [reply docs/source/tutorials/conversational_memory_with_langchain.md:7]: Ack

File: docs/source/tutorials/sycamore-jupyter-dev-example.md (hunk 2 above)
Comment (jonfritz) [reply docs/source/tutorials/sycamore-jupyter-dev-example.md]: Ack

File: docs/source/tutorials/sycamore-jupyter-dev-example.md (hunk 3 above)
Comment (jonfritz) [reply docs/source/tutorials/sycamore-jupyter-dev-example.md]: Ack

File: docs/source/querying_data/demo_query_ui.md [hunk 11]
```diff
@@ -1,9 +1,9 @@
 ## Using the Demo Query UI
 
-Sycamore includes a simple UI to submit queries and display retreival-augmented generation (RAG) answers and hybrid search results. The UI is deployed in the conatiner `sycamore-demo-ui` and is accessed at `localhost:3000`. It is an easy way to test the answer quality from your Sycamore stack, and how iterating with your data preparation and enrichment affects the overal quality.
+Sycamore includes a simple UI to submit queries and display retreival-augmented generation (RAG) answers and hybrid search results. The UI is deployed in the conatiner `sycamore-demo-ui` and is accessed at `http://localhost:3000`. It is an easy way to test the answer quality from your Sycamore stack, and how iterating with your data preparation and enrichment affects the overal quality.
```
[the remark this answers is not in the document]
Comment (jonfritz) [reply docs/source/querying_data/demo_query_ui.md:3]: That code isn't merged yet. He may need to update the docs when it is.

**Comment by jonfritz** [comment]
@HenryL27 responded to your comments and made the updates.

**Review by HenryL27 — APPROVED** [review]

**Pull request merged**